tests/async/test_headful.py

```diff
@@ -34,6 +34,7 @@ async def test_headless_should_be_able_to_read_cookies_written_by_headful(
 ):
     if is_chromium and is_win:
         pytest.skip("see https://github.com/microsoft/playwright/issues/717")
+        return
```

Comment:
I need to add the return in the two cookie tests from yesterday. Good catch!

Reply:
Actually I'm not sure if we need it! It was flaky so I added it. But seems like its not needed: https://www.programcreek.com/python/example/25423/pytest.skip

Reply:
✨ Magic!